main.go

```diff
@@ -51,6 +64,27 @@ func queryJson(state string, search string) (val string) {
 	return val
 }
 
+func getServices() []string {
```

Comment:
I wonder if maybe the getServices function should live in the app.go file. Since it will only be used there.

And that way the function definition of processResources wouldn't need to accomodate that extra services argument.

Reply:
Sure, I can move getServices to app.go! Just to let you know the reason I put that function in main.go is to prevent it get called multiple times in a loop.